courses/free-courses-bn.md

```diff
@@ -203,6 +205,7 @@ Shikkhangon BD
 ### PHP
 
 * [PHP All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVCFLIrGA1GaxacvPTDQcsMV) - Stack Learner
+* [PHP8 Bangla Tutorial](https://www.youtube.com/playlist?list=PLbC4KRSNcMnqBfSoiU5TG7FF4FQmCpoSp) - Learn Hunter
```

Comment:
Don't invent titles! Use the title given by the resource.
fine to truncate the title, but please truncate from the back so that alphabetization will expose duplicates

Reply:
I have changed the title to given by the resources.